src/utils/region.ts

```diff
@@ -0,0 +1,9 @@
+import { Region } from 'types/general/region';
+
+export const queryBuilderAddRegion = (query: object | undefined, region?: Region) => {
```

Comment:
```suggestion
export const addRegionToQuery = (query: object | undefined, region?: Region) => {
```

ALTERNATIVE: addRegionToQueryIfNeeded

you decide :)